ci/pipelines/deploy-to-perf.yml

```diff
@@ -0,0 +1,161 @@
+    aws_access_key_id: ((readonly_access_key_id))
+    aws_secret_access_key: ((readonly_secret_access_key))
+    aws_session_token: ((readonly_session_token))
+    aws_role_arn: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
+    aws_ecr_registry_id: "((pay_aws_test_account_id))"
+    aws_region: eu-west-1
+
+resources:
+  - name: deploy-to-perf-pipeline-definition
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: master 
+      paths:
+        - ci/pipelines/deploy-to-perf.yml
+  - name: pay-ci
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: master
+  - name: pay-infra
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-infra
+      branch: master
+      username: alphagov-pay-ci
+      password: ((github-access-token))
+  - name: adminusers-ecr-registry-perf
+    type: registry-image
+    icon: docker
+    source:
+      repository: govukpay/adminusers
+      variant: perf
+      <<: *aws_test_config
+  - name: telegraf-ecr-registry-perf
+    type: registry-image
+    icon: docker
+    source:
+      repository: govukpay/telegraf
+      variant: perf
+      <<: *aws_test_config
+  - name: nginx-proxy-ecr-registry-perf
+    type: registry-image
+    icon: docker
+    source:
+      repository: govukpay/docker-nginx-proxy
+      variant: perf
+      <<: *aws_test_config
+  - name: slack-notification
+    type: slack-notification
+    source:
+      url: https://hooks.slack.com/services/((slack-notification-secret))
+
+resource_types:
+  - name: registry-image
+    type: registry-image
+    source:
+      repository: concourse/registry-image-resource
+      tag: "1.4.1"
+  - name: slack-notification
+    type: docker-image
+    source:
+      repository: cfcommunity/slack-notification-resource
+      tag: latest
+
+groups:
+  - name: adminusers
+    jobs:
+      - deploy-adminusers-to-perf
+  - name: update-deploy-to-perf-pipeline
+    jobs:
+      - update-deploy-to-perf-pipeline
+
+jobs:
+  - name: update-deploy-to-perf-pipeline
+    plan:
+      - get: deploy-to-perf-pipeline-definition
+        trigger: true
+      - set_pipeline: deploy-to-perf
+        file: deploy-to-perf-pipeline-definition/ci/pipelines/deploy-to-perf.yml
+
+  - name: deploy-adminusers-to-perf
+    serial: true
+    plan:
+      - get: adminusers-ecr-registry-perf
+        trigger: true
+      - get: nginx-proxy-ecr-registry-perf
+        trigger: true
+      - get: telegraf-ecr-registry-perf
+        trigger: true
+      - get: pay-infra
+      - get: pay-ci
+      - load_var: application_image_tag
+        file: adminusers-ecr-registry-perf/tag
+      - load_var: nginx_image_tag
+        file: nginx-proxy-ecr-registry-perf/tag
+      - load_var: telegraf_image_tag
+        file: telegraf-ecr-registry-perf/tag
+      - put: slack-notification
+        params:
+          channel: '#govuk-pay-announce'
```

Comment:
I think this should probably go to `#govuk-pay-activity` - just have the failures going to `announce`?

Reply:
Fixed